mvc/views/tag-helpers/intro.rst

```diff
@@ -1,8 +1,281 @@
+-----------------------------------------------
+
+Tag Helpers don't own the element they're associated with; they simply participate in the rendering of the element and content. ASP.NET `Web Server controls <https://msdn.microsoft.com/en-us/library/7698y1f0.aspx>`__ are declared and invoked on a page.
+
+`Web Server controls <https://msdn.microsoft.com/en-us/library/zsyt68f1.aspx>`__ have a non-trivial lifecycle that can make developing and debugging difficult. Web Server controls allow you to add functionality to the client Document Object Model (DOM) elements by using a client control. Tag Helpers have no DOM. Web Server controls include automatic browser detection and some have the ability to pass events from a nested control (such as a button in a table) to the container control.
+
+Multiple Tag Helpers can act on the same element (see `Avoiding Tag Helper conflicts <http://mvc.readthedocs.org/en/latest/views/tag-helpers/authoring.html#avoiding-tag-helper-conflicts>`__ ) while you typically can't compose Web Server controls.
+
+Tag Helpers can modify the tag and content of HTML elements, but they don't directly modify anything else on a page. Web Server controls can include CSS stylesheets and javascript libraries, and they can bring in other assets that might have side effects on your page.
+
+Web Server controls use type converters to convert strings into objects. With Tag Helpers, you work natively in C#, so you don't need to do type conversion. 
+
+Web Server controls use `System.ComponentModel <https://msdn.microsoft.com/en-us/library/system.componentmodel%28v=vs.110%29.aspx>`__ to implement the run-time and design-time behavior of components and controls. ``System.ComponentModel`` includes the base classes and interfaces for implementing attributes and type converters, binding to data sources, and licensing components. Contrast that to Tag Helpers, which typically derive from ``TagHelper``, and the ``TagHelper`` base class exposes only two methods, ``Process`` and ``ProcessAsync``.
+
+Customizing the Tag Helper element font
+---------------------------------------------
+
+You can customize the font and colorization from **Tools** > **Options** > **Environment** > **Fonts and Colors**:
+
+.. image:: intro/_static/fontoptions.png
+
+.. TODO Move "Tag Helper naming conventions" to the authoring TH doc.
+
+Tag Helper naming conventions
+---------------------------------------
+
+We recommend, for Tag Helpers that target `existing HTML elements <http://www.w3.org/TR/html-markup/elements.html>`_, that you preface attribute names with a prefix that indicates the attribute is additive and run on the server. (We say additive because the server adds to the existing markup.) For example, the built-in Tag Helpers use the "asp-" prefix. The "asp-" prefix is not considered a reserved prefix, so developers can copy that convention. Some teams will prefer to use their own naming convention to distinguish their Tag Helpers.
```

Comment:
`attribute names with a prefix that indicates the attribute is additive and run on the server` =>
`additive attribute names with a prefix to indicate they are run on the server`